optaplanner-core/src/main/java/org/optaplanner/core/impl/partitionedsearch/scope/PartitionChangeMove.java

```diff
@@ -85,13 +86,13 @@ protected void doMoveOnGenuineVariables(ScoreDirector<Solution_> scoreDirector)
     }
 
     @Override
-    public boolean isMoveDoable(ScoreDirector<Solution_>  scoreDirector) {
+    public boolean isMoveDoable(ScoreDirector<Solution_> scoreDirector) {
         return true;
     }
 
     @Override
-    public PartitionChangeMove<Solution_> createUndoMove(ScoreDirector<Solution_> scoreDirector) {
-        throw new UnsupportedOperationException();
+    public NoChangeMove<Solution_> createUndoMove(ScoreDirector<Solution_> scoreDirector) {
+        return new NoChangeMove<>();
```

Comment:
Ow damn, I broke Partitioned Search without realizing it because we clearly have no test coverage for it.